adapters/LiftoffMonetize/LiftoffMonetizeAdapter/GADMAdapterVungleUtils.m

```diff
@@ -137,13 +72,24 @@ GADAdSize GADMAdapterVungleAdSizeForAdSize(GADAdSize adSize) {
     }
   } else if (size.height == GADAdSizeLeaderboard.size.height) {
     return GADAdSizeLeaderboard;
-  } else if (size.height == GADAdSizeMediumRectangle.size.height) {
-    return GADAdSizeMediumRectangle;
   }
-
   return GADAdSizeInvalid;
 }
 
+BannerSize GADMAdapterVungleConvertGADAdSizeToBannerSize(GADAdSize adSize) {
+  if (GADAdSizeEqualToSize(adSize, GADAdSizeMediumRectangle)) {
+    return BannerSizeMrec;
+  }
+  if (adSize.size.height == GADAdSizeLeaderboard.size.height) {
+    return BannerSizeLeaderboard;
+  }
+  // Height is 50.
```

Comment:
Can you clarify why is that the case?

Reply:
It is the only supported size from Vungle. Our SDK will try to fit the banner in the view, but the true height size of the assets is 50 px